azure-pipelines.yml

```diff
@@ -11,40 +11,52 @@ jobs:
   - script: |
       pip install -U docker-compose conan conan-package-tools
       cd modern && python run.py
+      pip uninstall --yes conan conan-package-tools
+      ./modern/run_pytest.sh
```

Comment:
Let's get rid of the logic hidden in Python scripts, too much magic. This `sh` could be copy/pasted here if needed. Just use plain commands that are easier to understand.